data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -619,16 +619,6 @@ def transform(self) -> None:
             lower=0
         )
 
-        # All values should have a value at this point
```

Comment:
I think we can get rid of this because we might not keep this filling all values, in line with what we see in QA